examples/config.yml

```diff
@@ -86,6 +86,15 @@ listen:
   #read_buffer: 10485760
   #write_buffer: 10485760
 
+# Routines is the number of thread pairs to run that consume from the tun and UDP queues.
+# Currently, this defaults to 1 which means we have 1 tun queue reader and 1
+# UDP queue reader. Setting this above one will set IFF_MULTI_QUEUE on the tun
+# device and SO_REUSEPORT on the UDP socket to allow multiple queues.
+#
+# You can also set this to the special string "auto" to pick a good default for
+# the number of CPUs you have. The current logic is (NumCPU - 1) / 2.
```

Comment:
We should point out that the minimum value is 1 with `auto`. Also maybe `0` == `auto`?